internal/common/model.go

```diff
@@ -2,7 +2,7 @@ package common
 
 const (
 	// BasePrefix is the base prefix for all keys
-	BasePrefix = "registry/"
+	BasePrefix = "metro/registry/"
```

Comment:
or just `metro/` ?

Reply:
Open to suggestions on this. @Abhishekvrshny

Reply:
just metro is fine here